src/components/auth/EmailLoginForm.tsx

```diff
@@ -46,43 +45,44 @@ export const EmailLoginForm = () => {
   }
 
   return (
-    <div className="flex flex-col gap-4">
+    <VStack>
       {/* Button to open the form on mobile view */}
-      {!open && (
-        <Button
-          large
-          onClick={openEmailLogin}
-          className="lg:hidden" // Show only on mobile view
-        >
-          Login with Email
-        </Button>
-      )}
+      <Button
+        onClick={openEmailLogin}
+        colorScheme="primary"
+        display={{ base: !open ? 'grid' : 'none', lg: 'none' }}
+      >
+        Login with Email
+      </Button>
 
       {/* Form, hidden on mobile view by default and shown only when open */}
-      <div className={cn({ block: open, hidden: !open }, 'lg:block')}>
-        <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col gap-4">
-          <span className="text-center text-foreground">Login to your account</span>
-          <TextInput
-            autoFocus
-            className="text-foreground"
-            label="Email"
-            type="email"
-            isError={!!errors.email}
-            {...register('email')}
-          />
-          <TextInput
-            className="text-foreground"
-            label="Password"
-            type="password"
-            isError={!!errors.email}
-            errorMessage={errors.email?.message}
-            {...register('password')}
-          />
-          <Button large type="submit" disabled={buttonDisabled}>
-            Login with Email
-          </Button>
-        </form>
-      </div>
-    </div>
+      <VStack
+        as="form"
+        onSubmit={handleSubmit(onSubmit)}
+        display={{ base: open ? 'flex' : 'none', lg: 'flex' }}
+        gap={4}
+      >
+        <Text textAlign="center">Login to your account</Text>
+
+        <TextInput
```

Comment:
I know they didn't do this, but I think it looks better if you do
```suggestion
        <Heading as="h2" fontSize="2xl" textAlign="center">
          Login to your account
        </Heading>

        <TextInput
```
![image](https://github.com/user-attachments/assets/b70eca18-e1cc-4250-b5ee-9afacc9ca1e5)